flowmachine/flowmachine/core/query.py

```diff
@@ -997,7 +997,7 @@ def random_sample(self, sampling_method="system_rows", **params):
         fraction : float, optional
             Fraction of rows to draw.
             Exactly one of the 'size' or 'fraction' arguments must be provided.
-        estimate_count : bool, default True
+        estimate_count : bool, default False
```

Comment:
What's the reason for changing this default? (Just so we have a record of the decision)

Reply:
Actually we could probably revert this, but with estimates on, you potentially under sample the higher end of the table, or return fewer than expected rows. Safer default.